InscryptionAPI/Saves/ModdedSaveManager.cs

```diff
@@ -40,11 +49,15 @@ public static void ReadDataFromFile()
             isSystemDirty = false;
         }
 
-        if (File.Exists(saveFilePath))
+        var oldSaveFileExist = File.Exists(oldSaveFilePath);
+
+        // If old save file exists, move it to new save file location
+        if (File.Exists(saveFilePath) || oldSaveFileExist)
         {
+            if (oldSaveFileExist) File.Move(oldSaveFilePath, saveFilePath);
```

Comment:
This will throw an exception if there is a save file in both locations. Either don't move the old file if a new one exists, or use the overload of File.Move that allows you to specify that you want to overwrite the target file.

Reply:
There's no overload for overwriting the target file, so i will go with the first option.

Reply:
On the last minute, I decided to make it check for the new file, and if both old and new file exist, delete the new one and move the old one to the new path.